coderd/provisionerdserver/provisionerdserver.go

```diff
@@ -1733,6 +1744,21 @@ func (s *server) CompleteJob(ctx context.Context, completed *proto.CompletedJob)
 		if err != nil {
 			return nil, xerrors.Errorf("update workspace: %w", err)
 		}
+
+		if input.PrebuildClaimedByUser != uuid.Nil {
```

Comment:
This feels narrow and fragile that we need to pass this imperative direction to provisionerdserver to send a signal specifically for prebuilds. It seems like a generally useful signal to know that a build completed for a given workspace... why not just always send it?

Even if we wanted to send the new owner ID in the message, there is no need to get it from the job input, we've already queried the workspace.